app/src/main/java/fr/free/nrw/commons/LocationPicker/LocationPickerActivity.java

```diff
@@ -405,6 +433,16 @@ void placeSelected() {
         setResult(AppCompatActivity.RESULT_OK, returningIntent);
         finish();
     }
+    /**
+     * Centre the camera on the last saved location
```

Comment:
In the rest of the project we use US spelling, would you mind changing this and other occurrences to "center"? Thanks!

Reply:
Would you mind fixing the "Centre" word in the javadoc comment above? Also in centre_on_gps and getCentre.